Binance.Net/BinanceClient.cs

```diff
@@ -1409,16 +1501,218 @@ public async Task<WebCallResult<object>> StopUserStreamAsync(string listenKey)
+        /// <returns>Transaction Id</returns>
+        public WebCallResult<BinanceRepay> Repay(string asset, decimal amount, int? receiveWindow = null) => RepayAsync(asset, amount, receiveWindow).Result;
+
+        /// <summary>
+        /// Repay loan for margin account.
+        /// </summary>
+        /// <param name="asset">The asset being repay, e.g., BTC</param>
+        /// <param name="amount">The amount to be borrow</param>
+        /// <param name="receiveWindow">The receive window for which this request is active. When the request takes longer than this to complete the server will reject the request</param>
+        /// <returns>Transaction Id</returns>
+        public async Task<WebCallResult<BinanceRepay>> RepayAsync(string asset, decimal amount, int? receiveWindow = null)
+        {
+            var timestampResult = await CheckAutoTimestamp().ConfigureAwait(false);
+            if (!timestampResult.Success)
+            {
+                return new WebCallResult<BinanceRepay>(timestampResult.ResponseStatusCode, timestampResult.ResponseHeaders, null, timestampResult.Error);
+            }
+
+            var parameters = new Dictionary<string, object>
+            {
+                { "asset", asset },
+                { "amount", amount.ToString(CultureInfo.InvariantCulture) },
+                { "timestamp", GetTimestamp() }
+            };
+            parameters.AddOptionalParameter("recvWindow", receiveWindow?.ToString(CultureInfo.InvariantCulture) ?? defaultReceiveWindow.TotalMilliseconds.ToString(CultureInfo.InvariantCulture));
+
+            return await ExecuteRequest<BinanceRepay>(GetUrl(MarginRepayEndpoint, MarginApi, MarginVersion), PostMethod, parameters, true).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Margin account new order
+        /// </summary>
+        /// <param name="symbol">The symbol the order is for</param>
+        /// <param name="side">The order side (buy/sell)</param>
+        /// <param name="type">The order type</param>
+        /// <param name="timeInForce">Lifetime of the order (GoodTillCancel/ImmediateOrCancel/FillOrKill)</param>
+        /// <param name="quantity">The amount of the symbol</param>
+        /// <param name="price">The price to use</param>
+        /// <param name="newClientOrderId">Unique id for order</param>
+        /// <param name="stopPrice">Used for stop orders</param>
+        /// <param name="icebergQty">Used for iceberg orders</param>
+        /// <param name="orderResponseType">The type of response to receive</param>
+        /// <param name="receiveWindow">The receive window for which this request is active. When the request takes longer than this to complete the server will reject the request</param>
+        /// <returns>Id's for the placed order</returns>
+        public WebCallResult<BinancePlacedOrder> MarginPlaceOrder(string symbol,
```

Comment:
```suggestion
        public WebCallResult<BinancePlacedOrder> PlaceMarginOrder(string symbol,
```

Reply:
Yes, I will add other methods today or tomorrow.
Thanks for a quick update.